helpers/constants.py

```diff
@@ -113,3 +113,13 @@
 module_new_draft_version = "at-uploader | Module new draft version test"
 assembly_new_draft_version = "at-uploader | Assembly new draft version test"
 
+#path for modules to unpublish
+module_to_unpublish = "content/repositories/uploader-test/entities/enterprise/modules/cockpit/pretty-host-name-in-the-web-console.adoc"
```

Comment:
Instead of hard coding these paths we can read the path from the document details page using the method display_module_page.get_path_to_adoc()